src/style.css

```diff
@@ -0,0 +1,117 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+  
+.form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+  
+.form--big {
+  width: 100%;
+  height: 70px;
```

Comment:
The height property should be defined by content height, we shouldn't set it in this case